influxdb/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.7"
+
+services:
+  
+  app_proxy:
+    environment:
+      APP_HOST: influxdb_influxdb_1
+      APP_PORT: 8886
+      PROXY_AUTH_WHITELIST: "/api/*"
+      
+  influxdb:
+    image: influxdb:1.11.7@sha256:b5efc528ee424e37716ce4596d32ad74cb595a4014741b25860e24feacb9868f
+    user: 1000:1000
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/data/data:/var/lib/influxdb
+      - ${APP_DATA_DIR}/data/config:/etc/influxdb
+      - ${APP_DATA_DIR}/data/influxdb:/.influxdb
+    environment:
+      V1_AUTH_USERNAME: admin
+      V1_AUTH_PASSWORD: admin
+    ports:
+      - '8089:80' # Web interface
+      - '8088:8088' # RPC service for backup and restore
```

Comment:
As far as I can tell, this version of influxdb doesn't have a web interface. I quickly tried installing this as-is and nothing shows up.

The container logs show that it has an api service listening on port `8086` though
```
$ sudo docker logs influxdb_influxdb_1
...
ts=2024-11-21T06:01:06.151944Z lvl=info msg="Starting HTTP service" log_id=0s~KnyLl000 service=httpd authentication=false
ts=2024-11-21T06:01:06.151949Z lvl=info msg="opened HTTP access log" log_id=0s~KnyLl000 service=httpd path=stderr
ts=2024-11-21T06:01:06.152025Z lvl=info msg="Listening on HTTP" log_id=0s~KnyLl000 service=httpd addr=[::]:8086 https=false
```

If I reconfigure to expose 8086 I can successfully ping the API.